src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php

```diff
@@ -118,7 +118,7 @@ protected function doSave(array $values, $lifetime)
             $ok = $this->write($file, '<?php return '.var_export($data, true).';') && $ok;
 
             if ($allowCompile) {
-                @opcache_compile_file($file);
+                @opcache_invalidate($file, true);
```

Comment:
Just verified with @jpauli: opcache_compile_file does not invalidate. Let's call only opcache_invalidate.